framework/db/Schema.php

```diff
@@ -478,7 +478,11 @@ public function quoteValue($str)
      */
     public function quoteTableName($name)
     {
-        if (strpos($name, '(') !== false || strpos($name, '{{') !== false) {
+
+        if (strpos($name, '(') === 0 && strpos($name, ')') === strlen($name) - 1) {
```

Comment:
```suggestion
        if (strncmp($name, '(', 1) === 0 && substr($name, -1) === ')') {
```

Reply:
@rob006 I made a few speed tests and found that your variant works slowly for 5-15%. I was surprised but it's true.

Reply:
Hmm, `strpos()` indeed could be more efficient in case of short strings.